pom.xml

```diff
@@ -11,7 +11,7 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
-        <zookpeeper.version>3.5.7</zookpeeper.version>
+        <zookpeeper.version>3.6.0</zookpeeper.version>
```

Comment:
Dependency changes should be in a separate patch.
This pat lch is about the website